debian/control

```diff
@@ -19,9 +19,10 @@ Depends:
  poppler-utils,
  graphicsmagick,
  python3 (>= 3.7.0),
- python3-nautilus | python-nautilus,
  python3-click,
  python3-pillow,
  python3-tqdm,
  ${misc:Depends}
+Suggests:
+ python3-nautilus | python-nautilus,
```

Comment:
This in practice means it will not be installed anymore (nautilus itself doesn't pull in python3-nautilus). But, since `qubes-core-agent-nautilus` (which is documented on https://www.qubes-os.org/doc/templates/minimal/) does depend on python3-nautilus, it should be okay.